src/main/resources/application.yml

```diff
@@ -49,14 +49,11 @@ nakadi:
   zookeeper:
     kafkaNamespace:
     brokers: 127.0.0.1:2181
-#    exhibitor:
-#      brokers: 127.0.0.1,127.0.0.2
-#      port: 8181
     sessionTimeoutMs: 30000
     connectionTimeoutMs: 10000
   oauth2:
     mode: BASIC
-    adminClientId: nakadi
+    adminClientId: stups_nakadi
```

Comment:
I think we should not use real value here. We can set some dummy id here and then override it in private configuration.